packages/lib-react-components/src/ZooFooter/ZooFooter.js

```diff
@@ -12,23 +11,6 @@ import {
   SocialAnchor
 } from './components'
 
-export const StyledEasterEgg = styled(Image)`
```

Comment:
[minor][trivia?] Oh, this may be a reference to a hidden feature in the original Zooniverse-React-Component's footer: https://github.com/zooniverse/Zooniverse-React-Components/blob/main/src/components/layout/zoo-footer.jsx#L86
```
<Footer>
  ...
  <Image className="footer__easter-egg" src="https://s3.amazonaws.com/zooniverse-static/assets/penguin.png" alt="" />
  ...
</Footer>
```

This was a small easter egg that displayed a penguin when you moused over a specific empty space on the footer... but uhh... I can't seem to see it on www.zooniverse.org any more. Huh, that's weird, could've sworn that PFE still uses old ZRC.

Anyway, if you can't find any mention of penguin.png in our code, then this easter egg might've been accidentally obliterated in the transition to FEM, alas.